samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -5,6 +5,8 @@
 
   "ServicePort": "40443",
 
+  "UseHttp": false, // Set to true to use HTTP instead of HTTPS - Only for development purposes
```

Comment:
I'm defaulting to using HTTPS.  Is this cool?

Reply:
Sure - let's start with this.